challenges/functions.js

```diff
@@ -7,25 +7,42 @@
   * In the body of the function return the callback with the two parameters that you created
 */
 
+function consume(a,b,cb) {
+  return cb(a,b) 
+
+  
+}
+
 
 /* Step 2: Create several functions to callback with consume();
   * Create a function named add that returns the sum of two numbers
   * Create a function named multiply that returns the product of two numbers 
   * Create a function named greeting that accepts a first and last name and returns "Hello first-name last-name, nice to meet you!"
 */
 
+function add(a,b) {
+  return a + b;
+}
+
+function multiply(a,b) {
+  return a * b;
+}
+
+function greeting(first, last) {
+  return `Hello ${first} ${last}, nice to meet you!`
+}
 
 /* Step 3: Check your work by un-commenting the following calls to consume(): */
-// consume(2,2,add); // 4
-// consume(10,16,multiply); // 160
-// consume("Mary","Poppins", greeting); // Hello Mary Poppins, nice to meet you!
+console.log(consume(2,2,add)) // 4
+console.log(consume(10,16,multiply)) // 160
+console.log(consume("Mary","Poppins", greeting))// Hello Mary Poppins, nice to meet you!
 
 
 // ==== Closures ==== 
 
 // Explain in your own words why `nestedfunction()` can access the variable `internal`.
 
-// Explanation: 
+// Explanation: The internal variable is in the global environment 
```

Comment:
am glad you are trying to understand closures )
you always can go over TK again https://learn.lambdaschool.com/fsw/module/rec1oabmeosilo2yf
or find more info somewhere else (f/e here https://medium.com/dailyjs/i-never-understood-javascript-closures-9663703368e8) 
let me know if you need more resources )